libmscore/mscore.h

```diff
@@ -18,8 +18,8 @@
 
 namespace Ms {
 
-#define MSC_VERSION     "3.01"
-static constexpr int MSCVERSION = 301;
+#define MSC_VERSION     "3.02"
+static constexpr int MSCVERSION = 302;
```

Comment:
Dies this impose a compatibility issue for score created with this version when opening them with older 3.x versions?

Reply:
Yes, it does. And I'm not sure that's what we want.

Reply:
I think we explicitly do **not** want this

Reply:
My idea too. Also, in the 3.x series we already introduced new tag which will be lost when opening/saving the score in an older 3.x version. It seems inconsistent to change that behavior now.

Reply:
Loosing a tag when opening/saving with a downrev version is the lesser evil here IMHO